federated_aws_cli/static/index.js

```diff
@@ -28,7 +28,7 @@ const selectRole = async (e) => {
 };
 
 const showRoles = async (roles, message) => {
-    if (roles.length === 0) {
+    if (roles["roles"].length === 0) {
```

Comment:
I might change this name a bit to be a little clearer? Not a big deal though.

Reply:
I was thinking the same. Hmm. Lemme see if I can think of a more clear way to do this.

Reply:
I've changed this in 1b6a809ea64eefdbc049422fff1a1d1c37b7f384